monitoring/README.md

```diff
@@ -58,12 +60,14 @@ thanos-storegateway-0                                  1/1     Running   0
 ```
 
 
-b) Deploy the monitoring tools in a WEC cluster (e.g., `cluster1`):
+b) Deploy the monitoring tools in a plain K8s WEC cluster (e.g., `cluster1`):
```

Comment:
I see `hub_context` being used in the plain kube case too, on lines 199 and 200 of the current edition of the script. They read as follows.

```shell
  export ROOT_USER=$(kubectl --context $hub_context get secret -n $ns minio -o jsonpath="{.data.rootUser}" | base64 -d)
  export ROOT_PASSWORD=$(kubectl --context $hub_context get secret -n $ns minio -o jsonpath="{.data.rootPassword}" | base64 -d)
```

Reply:
My point here is that the instructions here do NOT say to provide `--hub-context` in the plain kube case, but it seems like `--hub-context` is needed in the plain kube case too.

Reply:
The `--hub-context` flag has a default value that is used for the plain kube case. That's why that flag is omitted for the kube case instructions.

Reply:
That default value is "kind-kubeflex", right? Does the `monitoring/install-ks-monitoring.sh` script only work for OCP and the kind-based demo environment? If it works for cases where the KubeFlex hosting cluster is plain kube but not created by `kind` in our usual way then this should be documented. And if it is so restricted, that should also be documented.

Reply:
Ok, I see! The script `monitoring/install-ks-monitoring.sh` is not restrictive. It works too for any plain k8s environment besides kind. To make this clear, I will add the in the instructions for the plain kube case too.